### DISCUSSION TITLE
Semaphore Alternative

### DISCUSSION THREAD

**Discussion by ansibleguy** [semaphoreui/semaphore#1763] [Show and tell]
Greetings!

I like the idea of a lightweight WebUI for using Ansible.

As I am not successfully able to contribute to this project [because of some issues it has](https://github.com/ansible-semaphore/semaphore/discussions/1111) - I decided to create a similar solution from scratch.

* Ansible Web-UI: https://github.com/ansibleguy/ansible-webui
   Demo: https://demo.webui.ansibleguy.net/ (User: `demo`, Password: `Ansible1337`)
   Docs: https://ansible-webui.readthedocs.io/

It is still in early development, but the basic framework to run jobs, view job logs and multi-user setups incl. permissions & credentials are in place.

I would really love to get some feedback from interested testers. (:

Have a great day
\- AnsibleGuy

**Comment by tboerger** [comment]
I personally dislike that there is just another project... 

Generally it should be better to improve sensor. For my initial contributions I had the feeling that I have been ignored, but that has improved.

Without knowing further plans of @fiftin I would guess that he would be open to invite people into the org after they have shown that it is worth it, as it's the case for many opensource projects.

Even if the project really tends to fall behind because the only maintainer vanishes there is still time to properly fork the project, I have seen the same for Gitea which forked from Gogs.

So far I would say that improvements are welcome by @fiftin, so there is at least for me no interest in moving away from semaphore to a project which is written in python while I prefer simple Go projects ;)

**Reply by ruslan-mogilevskiy** [reply]
Is the Semaphore alive at all?
The issues list just growing here, the PRs aren't approved, the discussion about Semaphore's future mentioned above is stuck with no response from @fiftin.

**Reply by tboerger** [reply]
You can see on the pull request history and release history that the project is still alive.
